MapboxDirections/MBRouteStep.swift

```diff
@@ -557,6 +597,7 @@ open class RouteStep: NSObject, NSSecureCoding {
         let maneuverType = ManeuverType(description: maneuver["type"] as! String)
         let maneuverDirection = ManeuverDirection(description: maneuver["modifier"] as? String ?? "")
         let maneuverLocation = CLLocationCoordinate2D(geoJSON: maneuver["location"] as! [Double])
+        let roadSide = RoadSide(description: json["driving_side"] as! String) ?? .right
```

Comment:
Unsure how to handle a `nil` here.

Reply:
Is it possible for any production profile to not provide a `driving_side` property? If so, we would add an `unknown` enumeration value. If not, we would assert or implicitly unwrap the optional.